hal/include/HAL/Types.h

```diff
@@ -44,3 +44,13 @@ typedef HAL_Handle HAL_RelayHandle;
 typedef HAL_Handle HAL_SolenoidHandle;
 
 typedef int32_t HAL_Bool;
+
+#ifdef __cplusplus
+#define HAL_ENUM_START(name, type) enum name : type
+#define HAL_ENUM_END(name, type) ; /* NOLINT */
+#else
+#define HAL_ENUM_START(name, type) enum
```

Comment:
Since there's a typedef in HAL_ENUM_END that uses the enum name, shouldn't the name of the enum be specified here?